modules/distr-tracing-tempo-install-cli.adoc

```diff
@@ -0,0 +1,231 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-cli_{context}"]
+= Installing the {TempoShortName} by using the CLI
+
+You can install the {TempoShortName} from the command line.
+
+.Prerequisites
+
+* An active OpenShift CLI (`oc`) session by a cluster administrator with the `cluster-admin` role.
```

Comment:
Thank you so much. I didn't know that. As you can imagine, I'm new to the OpenShift docs (since two months and working on this PR only so far). I clearly need to catch up on onboarding. Also, it didn't occur to me to study the attributes list: will do so after this release! Thank you again.

Reply:
It's a relatively new attribute, it's not widely used because we haven't had time to sweep the repo for old instances yet. So, pretty easy to miss I think! :) I am super impressed at your first OCP docs PR - really legendary effort :D

Reply:
+1 to what Jeana said; legendary effort indeed!!

Reply:
Fixed in [this commit](https://github.com/openshift/openshift-docs/pull/62410/commits/dec1098fb34d993481085d6344ebd52a53013e29).